feature-interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt

```diff
@@ -152,15 +150,15 @@ private fun InterestsContent(
                     topics = uiState.topics,
                     onTopicClick = navigateToTopic,
                     onFollowButtonClick = followTopic,
-                    modifier = Modifier.padding(top = 8.dp)
+                    modifier = Modifier
```

Comment:
Nit: since it's customary for the default parameter for a modifier argument to just be `Modifier`, I think it's generally preferable to remove passing it at all and just use the default.